### PR TITLE
[Home Header] Improvements "Playlist"

### PR DESCRIPTION
**Pull request by Nanomani**
- display stat for playlist movies / playlist  TV Shows

### CODE REVIEW

**Comment by Nanomani** [comment]
Ups, there is a bug. Header moved to right a bit

File: 1080i/Includes_Defs.xml [hunk 1]
```diff
@@ -2971,6 +2971,17 @@
...
+	<include name="Header_PlayListStats">
+		<control type="list">
+			<visible>String.StartsWith(Container(300).ListItem.Property(path),ActivateWindow) + String.Contains(Container(300).ListItem.Property(path),Videos) + String.Contains(Container(300).ListItem.Property(path),playlists)</visible>
+			<width>1</width>
+			<height>1</height>
+			<focusedlayout />
+			<itemlayout />
+            <content>plugin://script.embuary.helper/?info=getpathstats&amp;path=$INFO[Container(300).ListItem.Property(path)]&amp;prefix=Playlist</content>
+		</control>
+	</include>
```
Comment (Nanomani) [inline 1080i/Includes_Defs.xml:2983]: This part is not good :/

**Comment by Nanomani** [comment]
@beatmasterRS 
For this one I have a problem.

There is no way for distinguish a "videos" playlist from a "musicvideos" playlist.
So I can't displayed the correct label "music videos" for a "musicvideos" playlist.

The best I can do is to add a condition with widget :  
` + String.Contains(Container(300).ListItem.Property(widgetType),musicvideos)`

This mean that works only if the user use an widget with a "musicvideos" playlist
otherwise the label will be "movies" not "musicvideos"

I have looked in the other skin (Amber, aeon nox silvo). All use script.skinshortcuts  : 
- for each main menu the user choose or not his "home header"
- this set a property. eg `<property name="InfoLine">musicvideoplaylist</property>`
- then it easy to test ` <value condition="String.IsEqual(Container(300).ListItem.Property(InfoLine),musicvideoplaylist)">......</value>`

Anyway nothing urgent and I don't think so I can more.
I will have tried :)

**Comment by beatmasterRS** [comment]
If i set "playlists" as a main menu entry, there seems to be an issue with the statistics. The count is releated to the playlist-count. So if i have 2 user generated playlists, 2 will be shown. But the label is wrong:

screenshot00491

If i select the playlists entry:

screenshot00495

And then go back to home:

screenshot00496

**Comment by beatmasterRS** [comment]
If i set "playlists" as a main menu entry, there seems to be an issue with the statistics. The count is releated to the playlist-count. So if i have 2 user generated playlists, 2 will be shown. But the label is wrong:

screenshot00491

If i select the playlists entry:

screenshot00495

And then go back to home:

screenshot00497

**Comment by Nanomani** [comment]
> If i set "playlists" as a main menu entry, there seems to be an issue with the statistics. The count is releated to the playlist-count. So if i have 2 user generated playlists, 2 will be shown. But the label is wrong:
> 
> If i select the playlists entry:
> 
> And then go back to home:
> 
I may not have understood. But clarification is needed. :)
When I say "playlists" I speak about user custom file Playlistfile.xsp :) not the main menu "playlists".
In this case it is not possible to have more one file UserPlaylistfile.xsp set to the main menu.

**Comment by beatmasterRS** [comment]
The issue i have mentioned, is releated to the statistics in the header. If i change a main menu entry to "playlists", then the header info is wrong. No matter, what widget content is set.

**Comment by Nanomani** [comment]
For you "playlists" is the menu who open windows with all list playlists.xsp ? right ?

**Comment by Nanomani** [comment]
Sorry I try to understand. I use only custom playlists in my side. I can't reproduce this. :/

**Comment by beatmasterRS** [comment]
Set "Playlists" as a main menu item:

screenshot00499

screenshot00501

screenshot00503

**Comment by Nanomani** [comment]
ok I understand now. :)
Thank you for the clarification.
this can't works and this is normal :D

you must use "playlist" not "playlists" like this :

**Comment by beatmasterRS** [comment]
Okay, so that means if I want to use the home menu header and set a main menu item "Playlists", I get statistics in the header that are not correct?

**Comment by Nanomani** [comment]
Yes you are right. 

I must add a new condition to exclude "playlists".
Then if possible add new code to displayed count item "playlist" if someone use "playlists".

Because it is impossible to have stats for movies / TVshow with "playlists". 
"playlists" is for displayed all custom "playlist" and then edit / add new "playlist".

I will make that soon

**Comment by Nanomani** [comment]
Must be better now : 



PS : sorry sometimes I quickly understand if someone explains to me for a long time :D

File: 1080i/Includes_Furniture.xml [hunk 2]
```diff
@@ -3166,6 +3170,8 @@
                 </control>
             </control>
             <control type="group">
+                <include>Header_PlaylistsCountItem</include>
```
Comment (beatmasterRS) [inline 1080i/Includes_Furniture.xml:3173]: Please add a condition to the indlude

**Comment by beatmasterRS** [comment]
Could you please test if everything is working properly?

**Comment by Nanomani** [comment]
> Could you please test if everything is working properly?

All is good for me :)

**Comment by beatmasterRS** [comment]
Thank you, will be merged soon :)

**Pull request merged**